core/controllers/collection_editor.py

```diff
@@ -221,10 +239,12 @@ def put(self, collection_id):
             else:
                 raise self.InvalidInputException(
                     'Cannot unpublish a collection.')
-
+        rights = rights_manager.get_collection_rights(
+            collection_id).to_dict()
+        rights['is_private'] = \
```

Comment:
Formatting note: we never have backslashes at the end of a line. Instead, put the "(" that starts the line below at the end of this line.